examples/producer-consumer/producer.pony

```diff
@@ -0,0 +1,23 @@
+use "collections"
+
+actor Producer
+	var _quantity_to_produce: U32
+	let _buffer: Buffer
+	var _num: U32 = 0
+	let _env: Env
+	
+	new create(quantity_to_produce: U32, buffer: Buffer, env: Env) =>
+		_quantity_to_produce = quantity_to_produce
+		_buffer = buffer
+		_env = env
+
+	be start_producing() =>
+		for i in Range[U32](0, _quantity_to_produce) do
```

Comment:
with high values for _quantity_to_produce, this is a GC antipattern.

GC won't happen for this actor until this method is finished running.
We'd want a comment to that effect or to change how this works to not trigger the gc anti-pattern.

I'd suggest doing in smaller batches and using a timer to trigger the batches based on having permission to produce. 
If you aren't familiar with how that would work, I'm happy to go over it on chat, skype/zoom/hangout or email.